lib/matplotlib/tests/test_axes.py

```diff
@@ -4972,3 +4972,23 @@ def test_bar_single_height():
     ax.bar(range(4), 1)
     # Check that a horizontal chart with one width works
     ax.bar(0, 1, bottom=range(4), width=1, orientation='horizontal')
+
+
+def test_invalid_axes_limits():
```

Comment:
I think you might have the 'side' and 'setter' swapped in the `parametrize` call.